packages/destination-actions/src/destinations/mantle/identify/index.ts

```diff
@@ -54,6 +54,34 @@ const action: ActionDefinition<Settings, Payload> = {
       default: {
         '@path': '$.traits.custom_fields'
       }
+    },
+    contactName: {
+      label: 'Contact Name',
+      description: 'The name of the contact / user',
+      type: 'string',
+      required: false,
+      default: { '@path': '$.traits.contact_name' }
+    },
+    contactEmail: {
+      label: 'Contact Email',
+      description: 'The email of the contact / user',
+      type: 'string',
+      required: false,
+      default: { '@path': '$.traits.contact_email' }
+    },
+    contactPhone: {
+      label: 'Contact Phone',
+      description: 'The phone number of the contact / user',
+      type: 'string',
+      required: false,
+      default: { '@path': '$.traits.contact_phone' }
```

Comment:
Phone numbers are usually send in the traits.phone location. 

```suggestion
      default: { '@path': '$.traits.phone' }
```